src/jit/lclvars.cpp

```diff
@@ -4118,8 +4127,17 @@ void Compiler::lvaComputeRefCounts(bool isRecompute, bool setSlotNumbers)
             // unreferenced later, we'll have to explicitly clear this bit.
             varDsc->setLvRefCnt(0);
             varDsc->setLvRefCntWtd(0);
-            varDsc->lvImplicitlyReferenced = 1;
-            varDsc->lvTracked              = 0;
+
+            // Special case for some varargs params ... these must
+            // remain unreferenced.
+            const bool isSpecialVarargsParam = varDsc->lvIsParam && raIsVarargsStackArg(lclNum);
+
+            if (!isSpecialVarargsParam)
+            {
+                varDsc->lvImplicitlyReferenced = 1;
+            }
```

Comment:
Is this where it is initialized? Does it need to be:
`varDsc->lvImplicitlyReferenced = isSpecialVarargsParam : 0 : 1;` ?

Reply:
The entire local var table is zero initialized when the table is allocated, so we don't need to set `lvImplicitlyReferenced` to zero. For instance, in `lvaInitTypeRef()`

```
    lvaTable         = getAllocator(CMK_LvaTable).allocate<LclVarDsc>(lvaTableCnt);
    size_t tableSize = lvaTableCnt * sizeof(*lvaTable);
    memset(lvaTable, 0, tableSize);
```